pyboy/core/ram.py

```diff
@@ -15,17 +16,20 @@
 
 
 class RAM:
-    def __init__(self, random=False):
-        if random: # NOTE: In real life, the RAM is scrambled with random data on boot.
-            raise Exception("Random RAM not implemented")
-
+    def __init__(self, randomize=False):
         self.internal_ram0 = array.array("B", [0] * (INTERNAL_RAM0))
         self.non_io_internal_ram0 = array.array("B", [0] * (NON_IO_INTERNAL_RAM0))
         self.io_ports = array.array("B", [0] * (IO_PORTS))
         self.internal_ram1 = array.array("B", [0] * (INTERNAL_RAM1))
         self.non_io_internal_ram1 = array.array("B", [0] * (NON_IO_INTERNAL_RAM1))
         self.interrupt_register = array.array("B", [0] * (INTERRUPT_ENABLE_REGISTER))
 
+        if randomize: # NOTE: In real life, the RAM is scrambled with random data on boot.
+            for mem in (self.internal_ram0, self.non_io_internal_ram0, self.io_ports, self.internal_ram1,
```

Comment:
[FEA0-FEFF range seems to not be randomized on DMG](https://gbdev.io/pandocs/#fea0-feff-range)
```suggestion
            for mem in (self.internal_ram0, self.internal_ram1):
```